src/usdb_syncer/usdb_scraper.py

```diff
@@ -369,7 +371,10 @@ def get_usdb_available_songs(
 
 
 def _parse_details_table(
-    details_table: BeautifulSoup, song_id: SongId, usdb_strings: Type[UsdbStrings]
+    details_table: BeautifulSoup,
+    song_id: SongId,
+    usdb_strings: Type[UsdbStrings],
+    logger: Log,
```

Comment:
Functions with a song id actually don't need to be passed a logger, as they can get it themselves: `get_logger(__file__, song_id)`.